requests/utils.py

```diff
@@ -696,6 +696,8 @@ def should_bypass_proxies(url, no_proxy):
 
     :rtype: bool
     """
+    # Prioritize lowercase environment variables over uppercase
+    # to keep a consistent behaviour with other http projects
```

Comment:
Would you mind specifying the two you checked (curl, wget) so we have some concrete examples?